### PR TITLE
Code Review at the Deadline

### PR DESCRIPTION
**Pull request by SamuelColacchia**

### CODE REVIEW

File: BTree.cc [hunk 1]
```diff
@@ -11,62 +11,623 @@
...
-bool BTree::lookup(string key, string & value) const
+
+/**
+ * @param string key | the key of the item to find
+ * @param BTreeFile::BlockNumber root | the current root
+ * @param string& value | the value we set if we find the key
+ *
+ * @description | Helper function called by lookup
+ */
+bool BTree::rlookup(string key, BTreeFile::BlockNumber currentBlockNumber, string value) const
```
Comment (russtuck) [inline BTree.cc:393]: Is this used?  I don't see a call in lookup().

**Comment by russtuck** [comment]
Passes all non-remove tests, plus simple remove from a leaf (5 pts extra credit).  Code is structured well and commented.

95/90

Sorry I don't have time to give more feedback.

File: BTree.cc [hunk 2]
```diff
@@ -11,62 +11,623 @@
  * Copyright (c) 2001, 2002, 2003 - Russell C. Bjork
  * Trivial changes 2016 - Russ Tuck
  */
```
Comment (russtuck) [inline BTree.cc:13]: Shouldn't you put your names here?

**Comment by SamuelColacchia** [comment]
Thanks for the feedback

**Pull request closed without merging**